python/composio/__init__.py

```diff
@@ -1,18 +1,31 @@
 # pylint: disable=wrong-import-position
 
+import json
+import sentry_sdk
+from pathlib import Path
 
 try:
-    import sentry_sdk
 
-    sentry_sdk.init(
-        dsn=(
-            "https://071d95df77abfdaaa87cd9441f42e12f@o4507825828462592.ingest.de."
-            "sentry.io/4507833628557392"
-        ),
-        traces_sample_rate=1.0,
-        profiles_sample_rate=1.0,
-        debug=False,
-    )
+    def _hook():
+        user_file = Path.home() / ".composio" / "user_data.json"
+        if not user_file.exists():
+            return
+
+        try:
+            json.loads(user_file.read_text(encoding="utf-8"))["api_key"]
+        except Exception:
+            return
+
+        sentry_sdk.init(
+            dsn=(
+                "https://071d95df77abfdaaa87cd9441f42e12f@o4507825828462592.ingest.de."
+                "sentry.io/4507833628557392"
+            ),
+            traces_sample_rate=1.0,
+            profiles_sample_rate=1.0,
+            debug=False,
+        )
+
 except Exception:
     pass
 
```

Comment:
It's good practice to handle exceptions more specifically rather than using a broad `except Exception`. Consider specifying the types of exceptions you expect to catch.